draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1489,14 +1489,27 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
 
 <section title="Considerations for New Header Fields" anchor="considerations.for.new.header.fields">
 <t>   
-   Authors of specifications defining new fields are
-   advised to keep the name as short as practical and not to prefix the name
-   with "X-" unless the header field will never be used on the Internet.
-   (The "X-" prefix idiom has been extensively misused in practice; it was
-   intended to only be used as a mechanism for avoiding name collisions inside
-   proprietary software or intranet processing, since the prefix would ensure
-   that private names never collide with a newly registered Internet name; see
-   <xref target="BCP178"/> for further information).
+   Authors of specifications defining new fields are advised to choose a short
+   but descriptive field name. Short names avoid needless data transmission
+   (especially when a header field is sent often); descriptive names avoid
+   confusion and "squatting" on names that might have broader uses.
+</t>
```

Comment:
Isn't that only true for HTTP/1.1?

Reply:
No; [HQ]Pack aren't magical, they have limited-size state tables that can be consumed by overly long header names.

Reply:
Yes, that's understood. But I think this is still will cause people tp be confused, in particular as it says "when ... sent often".

Reply:
OK, removing the parenthetical.